Ownership/Non-UK/Brest.txt

```diff
@@ -2,15 +2,11 @@ SECTOR:Brest PEMAK Triangle (245-660):24500:66000
 OWNER:BRN:BRL:BR:EW
 BORDER:SB1:LW15:LW16
 
-SECTOR:LFRR N:29500:66000
-OWNER:BRN:BRL:BR:EW
-BORDER:LFRR-N:LW17:LW16:LW6:LW7:LSRR:LFRR_LFFF
-
 SECTOR:LFRR Q:35500:66000
 OWNER:BRQ:BRE:BRL:BR:PW:P:EW
 BORDER:LSFF:LDM10:LDM9b:LFRR_LFEE:LFRR-E:LFRR_LFFF
 
-SECTOR:LFRR E:29500:35500
+SECTOR:LFRR E:25500:35500
```

Comment:
Hmmm, I think this is an error in the PDF we've been given. I'm going to find out whether it should be 295

Reply:
Now confirmed - Brest East sector starts at 295

Reply:
Have changed the level, I think Brest Q still works as the logon is Q?

Reply:
Ah right, yeah that's fine then!